os/drivers/Kconfig

```diff
@@ -431,7 +431,7 @@ if ANALOG
 source drivers/analog/Kconfig
 endif # ANALOG
 
-source drivers/kernel_test/Kconfig
+source drivers/os_api_test/Kconfig
```

Comment:
Please check the naming in below Kconfig also.

apps/examples/testcase/le_tc/kernel/Kconfig:	bool "Kernel TestCase Example"
apps/examples/testcase/le_tc/kernel/Kconfig:		Enable the Kernel TestCase Example
apps/examples/testcase/le_tc/kernel/Kconfig:	bool "Kernel Testcase Debug Output"